<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomePage</class>
 <widget class="QWidget" name="HomePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="TitleLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;电话计费管理系统&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddBtn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>添加会员</string>
   </property>
  </widget>
  <widget class="QPushButton" name="UserPayBtn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>会员缴费</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExitBtn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>退出系统</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ModifyBtn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>修改信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteBtn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除会员</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/backimage.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="QueryBtn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>查找会员</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>TitleLabel</zorder>
  <zorder>AddBtn</zorder>
  <zorder>UserPayBtn</zorder>
  <zorder>ExitBtn</zorder>
  <zorder>ModifyBtn</zorder>
  <zorder>DeleteBtn</zorder>
  <zorder>QueryBtn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
